hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseTokenHandler.java

```diff
@@ -156,6 +156,9 @@ protected void changeSupply(
         copyToken.totalSupply(newTotalSupply);
         copyTreasuryRel.balance(newTreasuryBalance);
 
+        // also need to update this
```

Comment:
Can we mark this TODO so we do not lose track?  This seems somewhat important to not forget.

Reply:
This comment is for the follow line which I added to fix one of the issues in the test. It's not a TODO.

Reply:
Ahh, it was a bit confusing.
Perhaps a tiny bit more detail (e.g. `We also need to update the Titles count on the copy`) would make the comment clearer.